src/search-input/search-input.js

```diff
@@ -0,0 +1,21 @@
+var React = require('react');
+
+var SearchInput = React.createClass({
+  propTypes: {
+    placeholder: React.PropTypes.string
+  },
+
+  render: function() {
+    var {placeholder, className, ...other} = this.props;
```

Comment:
Do we need to pass down placeholder? I thought react naturally passed down standard html attributes.